bin/launch_check_model.py

```diff
@@ -25,43 +25,7 @@ def get_args():
     args = parser.parse_args()
     return args
 
-"""
```

Comment:
why do you want to remove this class?

Reply:
it was the first prototype of the checkmodel wrote by mathys and it was commented for quite some PR by now. i commented at the beginning because i needed part of this for the new checkmodel class. but i forgot to remove it bck then afterwards.